### PR TITLE
Update UG with editm and deletem commands

### LINKED ISSUE #129: Update UG with editm and deletem commands

### PR DESCRIPTION
**Pull request by psh12320**
fixes #129

### CODE REVIEW

**Review by PeterHW963 — COMMENTED** [review]
Looks gay to me

File: docs/UserGuide.md [hunk 1]
```diff
@@ -157,12 +157,27 @@ Edits an existing Udder in the address book.
 
 Schedules a meeting with an Udder from the specified start time to end time, at the location.
 
-**Format:** `schedule UDDER_INDEX st/YYYY-MM-DD HH:MM et/YYYY-MM-DD HH:MM l/LOCATION`
+**Format:** `schedule UDDER_INDEX st/DD-MM-YYYY HH:MM et/DD-MM-YYYY HH:MM l/LOCATION`
```
Comment (XinghuaJulia) [inline docs/UserGuide.md:160]: Nice catch on formatting!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -180,6 +195,18 @@ Finds Udders by specified keywords for each field.
 
 ![result for 'find John'](images/findCommand3.png)
 
+### Delete meeting with an Udder: `deletem` 🗑️
```
Comment (XinghuaJulia) [inline docs/UserGuide.md:198]: I like the bin emoji used, very apt!

**Review by XinghuaJulia — COMMENTED** [review]
LGTM! Nice catch on formatting issues!

**Review by ulricolo7 — COMMENTED** [review]
LGTM

**Pull request merged**